src/measure_theory/measurable_space.lean

```diff
@@ -1075,3 +1076,58 @@ this.rec_on h_basic h_empty
   (assume f hf ht, h_union f hf $ assume i, by rw [eq]; exact ht _)
 
 end measurable_space
+
+namespace filter
+
+variables [measurable_space α]
+
+/-- A filter `f` is measurably generates if each `s ∈ f` includes a measurable `t ∈ f`. -/
+class is_measurably_generated (f : filter α) : Prop :=
```

Comment:
Could you mention this in the file-level docstring?

Reply:
Done.